File: app/src/main/java/com/fernandopretell/mymoments/PeruappsApplication.java
package com.fernandopretell.mymoments;

import android.content.Context;
import android.support.multidex.MultiDexApplication;

import timber.log.Timber;

public class PeruappsApplication extends MultiDexApplication {

    private static Context mContext;

    @Override
    public void onCreate() {
        super.onCreate();
        if (BuildConfig.DEBUG) {
            Timber.plant(new Timber.DebugTree());
        }
        mContext = getApplicationContext();
    }

    public static Context getAppContext() {
        return mContext;
    }

}

File: app/src/main/java/com/fernandopretell/mymoments/PeruappsApplication.kt
package com.fernandopretell.mymoments

import android.content.Context
import android.support.multidex.MultiDexApplication

import timber.log.Timber

class PeruappsApplication : MultiDexApplication() {

    override fun onCreate() {
        super.onCreate()
        if (BuildConfig.DEBUG) {
            Timber.plant(Timber.DebugTree())
        }
        appContext = applicationContext
    }

    companion object {

        var appContext: Context? = null
            private set
    }

}
